app/script.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+$(function() {
+
+  $('#name').on('click', function() {
+    $.get('adjective', function(response) {
+      var adjective = response.word;
+      $('#adjective').text(adjective);
+    });
+  });
+
+  $('#name').on('click', function() {
+    $.get('verb', function(response) {
+      var verb = response.word;
+      $('#verb').text(verb);
+    });
+  });
+
+  $('#name').on('click', function() {
+    $.get('noun', function(response) {
+      var noun = response.word;
+      $('#noun').text(noun);
+    });
+  });
+
+  $('#adjectiveButton').on('click', function() {
```

Comment:
good. Nice to see you taking something you've learned and using it again. Repetition will help you build skill. :)